backend/reservations/constants.py

```diff
@@ -0,0 +1,64 @@
+ROOM_LIST = {
+    'Queen': [
+        'Hard Rock - Standard 2 Queens (50% fee)',
+        'Hard Rock - Lakeview 2 Queens',
+        'Hard Rock - Lakeview 2 Queens (Post Sale)',
+        'Hard Rock - Lakeview Balcony 2 Queens',
+        "Bally's - Standard 2 Queens",
+        "Bally's - Standard 2 Queens (50% fee)",
+        "Bally's - Standard 2 Queens (Direct Sale)",
+        "Bally's - Standard 2 Queens (Post Sale)",
+        "Bally's - Standard 2 Queens (RFP Sale)",
+    ],
+
+    'Queen Sierra Suite': [
+        "Bally's - Sierra 2 Queens Suite",
+    ],
+
+    "King": [
+        'Hard Rock - Standard King',
+        'Hard Rock - Standard King (Post Sale)',
+        'Hard Rock - Standard King (RFP Sale)'
+        'Hard Rock - Lakeview King',
+        'Hard Rock - Lakeview Balcony King',
+        'Hard Rock - Lakeview Balcony King (Post Sale)',
+        'Hard Rock - Balcony King',
+        'Hard Rock - Balcony King (RFP Sale)',
```

Comment:
Could you clarify why these are in the same room list as the Bally's rooms? I'm not sure how thse are being used downstream, but seems like they might collide between the two venues somewhere...

Reply:
A Room is a Room is a Room. The distinction is handled based on the `name_hotel` field of the `Room` model.